src/interpreter/naive_interpreter.nit

```diff
@@ -71,6 +71,9 @@ class NaiveInterpreter
 	# The main Sys instance
 	var mainobj: nullable Instance is noinit
 
+        # Name of all supported functional names
```

Comment:
Weird indentation. Check your editor.

Reply:
I get it, you indent with spaces instead of tabs. This cause a misalignment in the diffs.
For most open-source projects, you should use the indentation convention of the files you are modifying.
Can you configure your editor to use tabs? http://nitlanguage.org/manual/syntax.html